src/pages/blog-posts.js

```diff
@@ -1,41 +1,51 @@
-import React from 'react'
-import { Link, graphql } from 'gatsby';
-import get from 'lodash/get'
-
-import BlogPost from '../components/BlogPost'
-
-
-class BlogPosts extends React.Component {
-  render() {
-    const posts = get(this, 'props.data.blogPosts.edges')
-
-    return (
-      <div>
-        {posts.map(post => (
-          <div>
-            <BlogPost key={post.node.slug} post={post.node} />
-            <Link to={`/${post.node.slug}`}>{post.node.title}</Link>
-            <br />
-          </div>
-        ))}
-        <Link to="/">Back to Home</Link>
-      </div>
-    )
-  }
+import PropTypes from "prop-types"
+import React from "react"
+import { Link, graphql } from "gatsby"
+
+import SimplifiedBlogPost from "../components/SimplifiedBlogPost"
+
+const linkStyle = {
+  margin: "20px",
+  textDecoration: "none",
+  color: "#f268ae",
+}
+
+const hrRule = {
+  width: "437px",
+  margin: "20px",
+}
+
+
+const BlogPosts = ({data}) => {
+  const posts = data.blogPosts.edges
+
+  return (
+    <div>
+      {posts.map(post => (
+        <>
```

Comment:
so this will raise a error because you are only adding a key to on elem -- `SimplifiedBlogPost`, whereas others are left without a key. I'd recommend wrapping this entire group into a div instead of a fragment `</>` and adding a key to it. What do you think?

Reply:
Checked the console, and you are correct errors abound!  Updating this :) Thanks for the heads up!